README.md

```diff
@@ -84,7 +84,7 @@ It's recommended to create an alias for a convenient usage:
 alias whipper="docker run -ti --rm --device=/dev/cdrom \
     -v ~/.config/whipper:/home/worker/.config/whipper \
     -v ${PWD}/output:/output \
-    whipper/whipper"
+    joelametta/whipper"
```

Comment:
This should probably be fixed by @JoeLametta making a "whipper" organisation and moving the docker image there, rather than fixing it in the README. :)